packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -1,9 +1,9 @@
 <template>
     <div
         ref="megaModal"
-        :class="['c-megaModal', $style['c-megaModal'], {
-            [$style['u-overlay']]: showOverlay
-        }]"
+        :class="['c-megaModal',
+                 $style['c-megaModal'],
```

Comment:
NIT: Do you need change the line-breaks here?

Reply:
eslint fixed it this way

Reply:
Weird, the style used elsewhere in the template is different.. Never mind though.